### ISSUE TITLE
Cypress Error - The plugins file is missing or invalid

### ISSUE THREAD

**Issue by Olubolar614** [TheBrainFamily/cypress-cucumber-preprocessor#386]
I'm getting error message "The plugins file is missing or invalid whenever I run tests in my local environment (Windows 10).   Link to repo - https://github.com/Olubolar614/olusimpletests
Error screenshot below

![CypressTest_Error](https://user-images.githubusercontent.com/26840342/83852146-4312b600-a70b-11ea-8773-115d66cafb2e.jpg)

**Comment by lgandecki** [comment]
you don't have cypress-cucumber-preprocessor defined in your devDependencies

**Comment by Olubolar614** [comment]
Hi Lukasz,

Thanks for the quick reply. Defining cypress-cucumber-preprocessor in
devDependencies (Package Json File) fixed the missing plugins file issue,
but when
I ran command "npx cypress run", I'm getting a second  error message "Could
not load reporter by name: cypress-multi-reporters".
Please find the attached error screenshot. I look forward to hearing from
you.

Kind regards

Olu

**Comment by lgandecki** [comment]
I'm sorry, this is a place for reporting issues with the library, not a place to ask for unpaid mentoring :-) as far as I see this is a fourth question that you ask on our plugin issue page and non of them were actual problems or feature requests.
Feel free to ask on stack overflow. At least let people gain some points for helping you. :)

**Comment by Olubolar614** [comment]
Hi Lukasz,

Your reply is not really professional and rude "Not a place to ask for
unpaid mentoring". If you cannot help people that are new to cypress, then
why create a cypress github repository?
I have the right to ask as many questions as possible, if you can't help,
then don't reply or keep your mouth shut.I hate people who can't deal with
people with respect.Lukasz, I don't know you from ADAM.

**Comment by badeball** [comment]
This is an *issue tracker*, and moreover a tracker for a very specific piece of code that merely does transformation. Your question has nothing to do with preprocessing anymore, hence this isn't the place to ask for help. We simply don't have the time to cater to your needs. The official [gitter chat](https://gitter.im/cypress-io/cypress) is usually a good place to start.

> If you cannot help people that are new to cypress, then why create a cypress github repository?

The repository exist simply for other to benefit from it and it is not an invitation for general purpose support.

Take your disgusting, garbage entitlement elsewhere.

**Comment by Olubolar614** [comment]
Hi Jonas

You are really a fool for using the phrase "Take your disgusting, garbage
entitlement elsewhere". I repeat, creating cypress github repository is an
opportunity
to support people that are just learning to use cypress. I don't care
whether you believe repository is an issue tracker, github is an open
source to support
developers and not too technical people. This is the last time I will reply
to your message because I don't have time for idiots like you.

**Comment by sztadii** [comment]
@Olubolar614 I understand you are very `not too technical people`, but I suggest different tone if you are expecting any help :)

**Comment by sztadii** [comment]
@Olubolar614 I understand you are very `not too technical people`, but I suggest a different tone if you are expecting any help :)

**Comment by Olubolar614** [comment]
Hi Krystian,

This is not a matter of not being too technical. I have been in the java
world for the past 12 years  and recently I was  just trying to upskill in
javascript and cypress.
My main point is the manner creator of this github repository
"TheBrainFamily/cypress-cucumber-preprocessor" reply to issue queries has
always been unprofessional.
Krystian, I hope you clearly understand my message.

Kind regards

Olu

**Comment by samhatoum** [comment]
Hey @Olubolar614 

For what it's worth, @lgandecki was not trying to be rude, he was saying that you're better off asking these questions in a place where you and other people can benefit. Github issues is for tracking issues.

As for this being a Cypress repository, actually it's not. It's a plugin that Lukasz wrote in his spare time and supports it out of his spare time. TheBrainFamily organization belongs to Lukasz, and he has created lots of open-source projects that many people use, and he supports them all for free, out of his spare time. 

So when you come here with a lot of questions that are not related to this plugin, but general Cypress questions, it's not always possible for him to help, especially when you are using this place as a Q&A. Github issues is not for Q&A, it's for *issues*. Places like StackOverflow are exactly for Q&A.

I'm sure you don't mean to take up people's time or to come across as entitled. Just keep this in mind when working with this or other repositories. Open source software takes a lot of time, and time is the most valuable currency anyone has so it's important be respectful with everyone's time.

I'll give you another lesson that I learned for free, is that different cultures communicate differently and sometimes what seems rude is more like a joke. This is especially true when you see lots of smiley faces as you saw in Lukasz's response. This indicates that he doesn't mean to be offensive and he's making a joke. 

II feel sorry for you that you come from a Java background :) I also came from Java. Keep at it, keep asking questions (in the right place) and you'll be answering questions yourself in no time.

Cheers 🍻
